### ISSUE TITLE
Chai-should fails when require without variable

### ISSUE THREAD

**Issue by saveman71** [skovhus/jest-codemods#172]
Repro:

```js
'use strict';

require('chai').should();

describe('Gift', () => {
    test('should create a gift', async () => {
      const savedDoc = await Gift.createGift();

      savedDoc.currency.should.equal('EUR');
    });
});
```

Produces:

```
repro.js Transformation error ([] does not match field "arguments": [Expression | SpreadElement] of type CallExpression)
Error: [] does not match field "arguments": [Expression | SpreadElement] of type CallExpression
    at addParam (/home/arch/.config/yarn/global/node_modules/jscodeshift/node_modules/ast-types/lib/types.js:565:23)
    at /home/arch/.config/yarn/global/node_modules/jscodeshift/node_modules/ast-types/lib/types.js:594:21
    at Array.forEach (<anonymous>)
    at Function.builder [as callExpression] (/home/arch/.config/yarn/global/node_modules/jscodeshift/node_modules/ast-types/lib/types.js:592:30)
    at NodePath.callExpression (/home/arch/.config/yarn/global/node_modules/jest-codemods/dist/transformers/chai-should.js:223:16)
    at NodePath.<anonymous> (/home/arch/.config/yarn/global/node_modules/jscodeshift/src/collections/Node.js:144:47)
    at __paths.forEach (/home/arch/.config/yarn/global/node_modules/jscodeshift/src/Collection.js:77:36)
    at Array.forEach (<anonymous>)
    at Collection.forEach (/home/arch/.config/yarn/global/node_modules/jscodeshift/src/Collection.js:76:18)
    at Collection.replaceWith (/home/arch/.config/yarn/global/node_modules/jscodeshift/src/collections/Node.js:142:17)
All done.
```

```js
const should = require('chai').should();
```

Fixes it.

**Comment by skovhus** [comment]
Thanks for contributing.

**Comment by skovhus** [comment]
Released as jest-codemods@0.21.0

https://github.com/skovhus/jest-codemods/releases/tag/0.21.0
